scripts/us_census/pep/us_pep_sex/process.py

```diff
@@ -0,0 +1,908 @@
+            'Age', 'Total', 'White Total', 'White Male', 'White Female',
+            'NonWhite Total', 'NonWhite Male', 'NonWhite Female'
+        ])
+        df['Year'] = year
+        df.insert(0, 'geo_ID', 'country/USA', True)
+        df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
+    else:
+        df = pd.read_csv(file_path,
+                         thousands=',',
+                         engine='python',
+                         skiprows=6,
+                         skipfooter=102,
+                         header=None)
+        df.columns = [
+            'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female',
+            'White Total', 'White Male', 'White Female', 'Black Total',
+            'Black Male', 'Black Female', 'OtherRace Total', 'OtherRace Male',
+            'OtherRace Female'
+        ]
+        df = df.drop(columns=[
+            'Age', 'Total', 'White Total', 'White Male', 'White Female',
+            'Black Total', 'Black Male', 'Black Female', 'OtherRace Total',
+            'OtherRace Male', 'OtherRace Female'
+        ])
+        df['Year'] = year
+        df.insert(0, 'geo_ID', 'country/USA', True)
+        df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
+    return df
+
+
+def _national_1990_2000(file_path: str):
+    """
+    Process and cleans the file for national 1990 to 2000.
+
+    Args:
+        file_path (str) : input file path.
+
+    Returns:
+        df (pd.DataFrame) : cleaned dataframe.
+    """
+    df = pd.read_csv(file_path, thousands=',', skiprows=1, header=None)
+    df.columns = [
+        'Year', 'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female'
+    ]
+    df = df[df['Year'].str.contains("July 1")]
+    df = df.query('Age=="All Age"')
+    df['Year'] = df['Year'].str.replace("July 1, ", "")
+    df = df.drop(columns=['Total', 'Age'])
+    df.insert(0, 'geo_ID', 'country/USA', True)
+    float_col = df.select_dtypes(include=['float64'])
+    for col in float_col.columns.values:
+        df[col] = df[col].astype('int64')
+    df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
+    return df
+
+
+def _national_2000_2010(file_path: str):
+    """
+    Process and cleans the file for national 2000 to 2010.
+
+    Args:
+        file_path (str) : input file path.
+
+    Returns:
+        df (pd.DataFrame) : cleaned dataframe.
+    """
+    df = pd.read_csv(file_path, thousands=',', skiprows=4, header=None)
+    df.columns = [
+        'SEX', 'April2000', '2000', '2001', '2002', '2003', '2004', '2005',
+        '2006', '2007', '2008', '2009', 'April2010', '2010'
+    ]
+    df = df.query('SEX=="MALE" or SEX=="FEMALE"')
+    df = df.drop(columns=['April2000', 'April2010', '2010'])
+    df = df.replace(
+        {'SEX': {
+            'MALE': 'Count_Person_Male',
+            'FEMALE': 'Count_Person_Female'
+        }})
+    df = df.transpose().reset_index()
+    df.columns = df.iloc[0]
+    df = df[1:]
```

Comment:
Can you provide some comments here... about functionality

Reply:
Added comments.